### PR TITLE
Blog shk

### PR DESCRIPTION
**Pull request by jcolomb**
add a blog post to link when looking for SHK

### CODE REVIEW

File: content/blog/2021-10-25-student-assistent-position-available/index.en.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - position
+banner: img/banners/hiring.png
+tags: [job]
+---
+
+In [Open.Make](/blog/2021/09/27/2021-09-27-scope/) we are looking for three motivated student assistants to support our research team and enrich it with your own unique and complementary skillset.
+
+Any student of a Berlin university can apply to any of the three positions (salary 12,68 € brutto according to TV Stud III). The majority of the work needs to be done on site. Yet, some of the work can also be carried out remotely.
+
+We will prioritize candidates who can make our team more diverse, in accordance with the universities' provisions. Especially women should feel addressed, too.
```
[the remark this answers is not in the document]
Comment (jcolomb) [reply content/blog/2021-10-25-student-assistent-position-available/index.en.md]: got closer to uni text

**Review by jcolomb — COMMENTED** [review]
now good to go I think

**Review by MoritzMaxeiner — APPROVED** [review]
lgtm

**Comment by jcolomb** [comment]
replaced by #12

**Pull request closed without merging**